src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class PostalCode {
+
+	public static final String EXAMPLE = "123456";
```

Comment:
indentation is off in multiple places here.
check how you can get the IDE to do this for you :-)